client/testFixture/one/imports.omt

```diff
@@ -1,7 +1,7 @@
 import:
     module:Declared:
     -   DeclaredQuery
-    ../relative.omt:
+   ../relative.omt:
```

Comment:
The YAML parser for OMT is probably not be okay with differently indented entries. The plugin should work just fine though.

Reply:
yes, and this line is tot test that the plugin works as intended.